docs/Introduction/whatIsLitProtocol.md

```diff
@@ -2,49 +2,122 @@
 sidebar_position: 2
 ---
 
-# Introduction
+# What is Lit Protocol?
```

Comment:
I would revert this back to # Introduction

Since it doesn't look at good with the double "What is Lit Protocol" in the side bar

<img width="287" alt="Screen Shot 2022-10-27 at 12 17 20 PM" src="https://user-images.githubusercontent.com/5404516/198379389-ee3e31e6-4e14-4f47-bbd7-2f72a0f47fcb.png">